src/components/modals/movieDetails/movieDetails.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+import styles from './movieDetails.module.css'
+
+interface MovieDetailsProps {
+  name: string
+  year: number
+}
+
+export const MovieDetails: React.FC<MovieDetailsProps> = ({ name, year }) => {
+  return (
+    <article>
+      <div className={styles.heading}>
+        <h2>netflixroulette</h2>
+      </div>
+      <div className={styles.mainSection}>
+        <img
+          className={styles.image}
+          src="../../../../assets/parazity.png"
```

Comment:
import it like this with name 
`import imgName from “../../../../assets/parazity.png”`

and then use 
`src={imgName}`

If import gives you an error - something wrong with webpack loader, check it